modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/channels/pages/__tests__/ChannelList.jsx

```diff
@@ -62,4 +64,86 @@ describe('Channels List', () => {
 
 		expect(queryByText('New Property')).toBeNull();
 	});
+
+	it('should check if the number of synced sites and channels appears for each property', async () => {
+		API.channels.search.mockReturnValue(
+			Promise.resolve(data.mockChannels())
+		);
+
+		const {container, getByText} = render(
+			<Provider store={mockStore()}>
+				<MemoryRouter
+					initialEntries={['/workspace/23/settings/properties']}
+				>
+					<Route path={Routes.SETTINGS_CHANNELS}>
+						<ChannelList {...defaultProps} />
+					</Route>
+				</MemoryRouter>
+			</Provider>
+		);
+
+		jest.runAllTimers();
+
+		await waitForLoadingToBeRemoved(container);
+
+		expect(getByText('Sites')).toBeInTheDocument();
+
+		expect(getByText('Channels')).toBeInTheDocument();
+
+		expect(getByText('Liferay DXP')).toBeInTheDocument();
+
+		const sitesCount = getByText('6');
+
+		const channelsCount = getByText('5');
+
+		expect(sitesCount).toBeInTheDocument();
+
+		expect(channelsCount).toBeInTheDocument();
+	});
+
+	it('should check if actions "DELETE" and "CLEAR DATA" are displayed in the ellipsis and its icons', async () => {
+		API.channels.search.mockReturnValueOnce(
+			Promise.resolve(data.mockChannels())
+		);
+
+		const {container, getByText} = render(
+			<Provider store={mockStore()}>
+				<MemoryRouter
+					initialEntries={['/workspace/23/settings/properties']}
+				>
+					<Route path={Routes.SETTINGS_CHANNELS}>
+						<ChannelList {...defaultProps} />
+					</Route>
+				</MemoryRouter>
+			</Provider>
+		);
+
+		jest.runAllTimers();
+
+		await waitForLoadingToBeRemoved(container);
+
+		expect(getByText('Delete')).toBeInTheDocument();
+
+		expect(getByText('Clear Data')).toBeInTheDocument();
+
+		const clearDataBtn = container.querySelector(
+			'button[aria-label="Clear Data"]'
+		);
+
+		const deleteBtn = container.querySelector(
+			'button[aria-label="Delete"]'
+		);
+
+		expect(clearDataBtn).toBeInTheDocument();
+
+		expect(deleteBtn).toBeInTheDocument();
+
+		const clearDataIcon = clearDataBtn.querySelector('svg.icon-root');
+
+		const deleteIcon = deleteBtn.querySelector('svg.icon-root');
+
+		expect(clearDataIcon).toBeInTheDocument();
+
+		expect(deleteIcon).toBeInTheDocument();
```

Comment:
We need to improve the way to get icon here.

Reply:
@interaminense it's not possible to use `screen.getByRole` here because it will be confused with row actions buttons. So I tried a different approach. Please take a look and see if it's ok :-)